mtags/src/main/scala/scala/meta/internal/mtags/DefinitionAlternatives.scala

```diff
@@ -0,0 +1,86 @@
+package scala.meta.internal.mtags
+
+import scala.meta.internal.semanticdb.Scala._
+
+object DefinitionAlternatives {
+
+  /** Returns a list of fallback symbols that can act instead of given symbol. */
+  def apply(symbol: String): List[String] = {
+    List(
+      caseClassCompanionToType(symbol),
+      caseClassApplyOrCopy(symbol),
+      caseClassApplyOrCopyParams(symbol),
+      methodToVal(symbol),
+      methodOwner(symbol),
+    ).flatten
```

Comment:
In case this becomes a hot path, maybe using `++` instead of `List.apply` + `flatten` is slightly more conservative.

Reply:
I don't think this is a bottleneck since is not on a critical path, it's a fallback to missing symbols. If we want to squeeze out the last juice of performance we should benchmark it first. I'm also not sure if `List[List[T]].flatten` is slower than `++`